bettercli/cl/date.py

```
@@ -0,0 +1,316 @@
+            self.cache[year][month] = []
+            return []
+    
+    def get_or_add(self, month:'int', year:'int', func:'t.Callable[[int, int], list[list[d]]]'):
+        if year not in self.cache:
+            self.cache[year] = {}
+        if month not in self.cache[year]:
+            self.cache[year][month] = func(month, year)
+        return self.cache[year][month]
+
+class Dict(dict):
+    def __init__(self, dict:'dict'={}):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)